### PR TITLE
fix price bucket compute error

### PR DESCRIPTION
**Pull request by YongwenZhao**
error: no method to_d

### CODE REVIEW

File: core/app/models/calculator/price_buckets.rb [hunk 1]
```diff
@@ -15,13 +15,8 @@ def self.register
...
   # as object we always get line items, as calculable we have Coupon, ShippingMethod
   def compute(object)
-    if object.is_a?(Array)
-      base = object.map{ |o| o.respond_to?(:amount) ? o.amount : o.to_d }.sum
-    else
-      base = object.respond_to?(:amount) ? object.amount : object.to_d
-    end
-
-    if base >= self.preferred_minimal_amount
+    base = object.line_items.map(&:amount).sum
```
Comment (still0007) [inline core/app/models/calculator/price_buckets.rb]: Every object in ```line_items``` has the method ```amount```?
Why previous version has the check: ```o.respond_to?(:amount) ? o.amount : o.to_d```

Comment (YongwenZhao) [inline core/app/models/calculator/price_buckets.rb]: Yes, in shop here object is an order, every order line_item has amount. Maybe here the check is in order to use spree in other project.

Comment (still0007) [inline core/app/models/calculator/price_buckets.rb]: got it.

**Comment by still0007** [comment]
:+1

**Pull request merged**